content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Installing the Istio control plane
+
+1. Set the name of your cluster as an environment variable.
+
+    {{< text bash >}}
+    $ CLUSTER_NAME="<the name of your cluster>"
```

Comment:
It might be helpful to establish all of the variables up-front in a table form. This way there are no surprises as the user goes through the guide. See the [multciluster docs](https://docs.google.com/document/d/1wzoFqLHw-dEoKPVi1w5tlckWTn78qpYNbC5fEwp-Mw4/edit#heading=h.vu9te4khxjut)